tests/test.py

```diff
@@ -458,6 +458,9 @@ def run_cloudformation(*, image: str, user: str) -> None:
     secure_volumes: dict[Path, dict[str, str]] = {
         secure_test_dir: {"bind": working_dir, "mode": "rw"}
     }
+    alt_working_dir: str = "/alt_working_dir/"
```

Comment:
These tests are just a rough poc and need to be improved before merging.

Reply:
Okay so dug into this further; we hit an issue that is specific to bitbucket pipelines which we can't reasonably reproduce in a github action so I think having some baseline tests like this is good enough, and if we find a way to improve this we can handle it in the future.